### PR TITLE
added product2 page

### PR DESCRIPTION
**Pull request by sushmita12345**
Added product2 page to the E-commerce website.

Sample

### CODE REVIEW

**Review by palash23jain — APPROVED** [review]
looks very clean

File: components/product2.html [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+                                </div>
+                                <span class="discount">50% off</span>
+                            </div>
+                            <div class="card-button">
+                                <button class="card-btn">Go to cart</button>
+                                <button class="card-btn btn-color-addon">Remove from Wishlist</button>
+                            </div>
+                            
+                        </div>
+
```
Comment (PuneetP16) [inline components/product2.html]: spacing could be made even.

**Review by PuneetP16 — APPROVED** [review]
Good to merge after necessary changes

File: components/product2.html (hunk 1 above)
Comment (sushmita12345) [reply components/product2.html]: Ok sure, Thanks.

**Pull request merged**